src/app/applications/components/rollout-node-info/rollout-node-info.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import * as models from '../../../shared/models';
+import { ResourceTreeNode } from '../utils';
+
+export const RolloutNodeInfo = (props: { node: ResourceTreeNode, live: models.State}) => {
+    return (
+        <div className='white-box'>
+        <div className='white-box__details'>
+            {props.live.spec.strategy.blueGreen.previewService !== '' && (
```

Comment:
I think `props.live.spec.strategy.blueGreen` and `props.live.status` might be null . Please add null checks to avoid NPE